samples/misc/NodeServicesExamples/NodeServicesExamples.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Nit: likewise here, remove the default values.

(Repeat same comment for most of the projects below)